### DISCUSSION TITLE
Proteomics for turnover prediction

### DISCUSSION THREAD

**Discussion by soulios** [ProteomicsML/ProteomicsML#194] [Dataset discussions]
Hello to the community,

I have no previous experience with proteomics data( or-omics tbh) and I want to predict protein turnover with such data.
Most current methods use the protein sequence, and they rely on a substrate to calculate the Kcat.
Does it exist any dataset for this scenario? From  my understanding this should come from labeling and unlabeling SILAC experiments as mentioned [here](https://www.sciencedirect.com/science/article/pii/S1535947620334526).
But are these experiments common? Should I selHow one would go about it to construct such a dataset?

Thank you in advance.

**Comment by magnuspalmblad** [comment]
Indeed, isotopic labeling, either by adding labeled amino acids to the medium, as in SILAC, or their inorganic precursors, is commonly used in protein turnover experiments. A quick [search](https://proteomecentral.proteomexchange.org/ui?search=%22protein%20turnover%22) in ProteomeXchange suggests there are close to 100 such datasets. Even though all of these may be suitable for training a model predicting protein turnover, this is where I would start.

As an aside, for really long time scales (decades) and organisms living outside the lab, C-14 dating on the [bomb curve](https://en.wikipedia.org/wiki/Bomb_pulse) has been used to date [keratin and collagen](https://doi.org/10.1073/pnas.1302226110), for example.

Good luck with your predictions!
